src/app/features/animals/components/animals-form/animals-form.component.html

```diff
@@ -0,0 +1,21 @@
+<h1>Add animal</h1>
+<form (ngSubmit)="submitForm()">
+  <div class="mb-3">
+    <label class="form-label">Name</label>
+    <input type="text" name="name" class="form-control" [(ngModel)]="name">
```

Comment:
Pre vstupy si použil `ngModel` a chýba validácia vstupov. Namiesto `ngModel` je lepšie použiť reactive forms a k nim aj validátor.

```suggestion
    <input type="text" formControlName="name" class="form-control">
```

Reply:
Fixol som to a použil som reactive forms.